apps/emqx_ft/src/emqx_ft.erl

```diff
@@ -331,32 +337,13 @@ assemble(Transfer, FinalSize, FinalChecksum) ->
             {error, {internal_error, E}}
     end.
 
+assemble_kickoff(Pid) ->
+    erlang:send(Pid, kickoff).
```

Comment:
Nit. Looks like it should be part of `emqx_ft_assembler` module now?

Reply:
✅ 
Moved to `emqx_ft_storage`, because it encapsulates what is behind the pids